packages/apps/test/nim.spec.ts

```diff
@@ -73,6 +72,9 @@ describe("Nim", () => {
   before(async () => {
     const provider = waffle.createMockProvider();
     const wallet = (await waffle.getWallets(provider))[0];
+    if (!NimApp.bytecode) {
+      throw Error("empty bytecode");
+    }
```

Comment:
Why is this necessary? The build-test flow is meant to handle this. An error of this sort should be thrown at the very beginning of the entire test process I think. Perhaps in the jest entrypoint file.